### DISCUSSION TITLE
Added OpenAi key to local setup , not working

### DISCUSSION THREAD

**Discussion by pranav-more** [mckaywrigley/chatbot-ui#1575] [Help]
I have followed all steps from readme and local setup is done with docker instance on supabase , i have added openai key locally 
but when I run locally and try to chat it says - 404 The model `gpt-4-turbo-preview` does not exist or you do not have access to it. 

<img width="1362" alt="Screenshot 2024-03-19 at 12 29 23 AM" src="https://github.com/mckaywrigley/chatbot-ui/assets/51186180/c039a06c-99aa-455f-8ca0-b0d809712059">

**Comment by pranav-more** [comment]
this was resolved

**Reply by MuhammedAYBU2020** [reply]
How did you solved this problem?

**Reply by nefariousOne** [reply]
> How did you solved this problem?

You'll have to add money to the API "platform" aspect of OpenAI, simply paying for a ChatGPT Plus subscription, doesn't allow for API access.

https://help.openai.com/en/articles/7102672-how-can-i-access-gpt-4-gpt-4-turbo-gpt-4o-and-gpt-4o-mini
https://platform.openai.com/settings/organization/billing/overview
